src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UpdatePreviewResult.cs

```diff
@@ -21,6 +22,10 @@ public override string ToString()
             return Old + " -> " + New;
         }
 
-        public string AutomationName => $"{Old} to {New}";
+        public string AutomationName => string.Format(
```

Comment:
See my other comment.  Can't we create this value once and only once?

Reply:
Sure.